package com.volodymyr.pletnov.test_task.exception;

public class NotFoundException extends RuntimeException{

	public NotFoundException(String message) {
		super(message);
	}
}
